packages/preview/hpi-thesis/0.0.1/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "hpi-thesis"
```

Comment:
Because this template is not official, can you please change its name to be less canonical, to follow [our naming guidelines](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#naming-rules).